stable/telegraf/values.yaml

```diff
@@ -0,0 +1,491 @@
+## Default values.yaml for Telegraf
+## This is a YAML-formatted file.
+## ref: https://hub.docker.com/r/library/telegraf/tags/
+image:
+  repo: "telegraf"
+  tag: "1.1.0-alpine"
+  pullPolicy: IfNotPresent
+
+## Configure the telegraf daemonset here.
+## Resource limits and outputs can be set seperately
+daemonset:
+  enabled: true
+  ## Configure resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  resources:
+    requests:
+      memory: 256Mi
+      cpu: 0.1
+    limits:
+      memory: 2Gi
+      cpu: 1
+  ## Exposed telegraf configuration
+  ## ref: https://docs.influxdata.com/telegraf/v1.1/administration/configuration/
+  config:
+##    global_tags:
+##      dc: "us-east-1"
+##      user: "$USER"
+    agent:
+      interval: "10s"
+      round_interval: true
+      metric_batch_size: 1000
+      metric_buffer_limit: 10000
+      collection_jitter: "0s"
+      flush_interval: "10s"
+      flush_jitter: "0s"
+      precision: ""
+      debug: false
+      quiet: false
+      logfile: ""
+      hostname: ""
+      omit_hostname: false
+    outputs:
+      influxdb:
+        urls:
+          - "http://influxdb-influxdb.tick:8086"
```

Comment:
Since this is a required value I would leave it unset and not deploy the daemonset or other deployment until it gets set explicitly by the user. Check out the Ghost chart for an example of that pattern.